inst/rmarkdown/templates/jss_article/skeleton/skeleton.Rmd

```diff
@@ -43,11 +43,20 @@ In general, don't use Markdown, but use the more precise LaTeX commands instead:
 
 One exception is inline code, which can be written inside a pair of backticks (i.e., using the Markdown syntax).
 
-# R code
+If you want to use LaTex commands in headers, you need to provide a `short-title`
+attribute. You can also provide a custom identifier if necessary.
+See the example of the header \ref{r-code}
```

Comment:
```suggestion
See the header of Section \ref{r-code} for example.
```

BTW, since other paragraphs are not wrapped, let's not wrap this one, either.

If you prefer hard wrapping, we can use the RStudio daily build with the visual editor, so you don't have to manually wrap the text.